src/components/POLine/Item/ErrorMessage.css

```diff
@@ -0,0 +1,4 @@
+.errorMessage {
+  margin-bottom: 1rem;
+  color: #900;
```

Comment:
I believe it makes sense to use CSS variables for these properties https://github.com/folio-org/stripes-components/blob/master/lib/variables.css#L29